sdk/identity/azure-identity/azure/identity/_credentials/linux_vscode_adapter.py

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+import json
+import ctypes as ct
+from .._constants import VSCODE_CREDENTIALS_SECTION
+
+
+def _c_str(string):
+    return ct.c_char_p(string.encode('utf-8'))
+
+
+try:
+    _libsecret = ct.cdll.LoadLibrary('libsecret-1.so.0')
```

Comment:
Do we want a message if the lib is not available, to do something like `sudo apt-get install libsecret-1-0` or something?

Reply:
In fact, this credential is usually used in DefaultCredential which is a chain of credentials. If one does not work, it will try next one. So if libsecret-1-0 is not installed, we prefer just skip this one and go to next credential rather than failing.

Reply:
In other words, if libsecret-1-0 is not installed, the feature is disabled silently.

Reply:
Re-thinking this, I guess if they have vscode the lib is available, or maybe vscode embed it to save their credentials?

Reply:
Azure connectors (e.g. azure-account, azure-storage, etc.) are vs code extensions. User can use VS code w/o them. Given this case, user uses vs code w/o Azure extensions. (and libsecret-1-0 is not installed). We should just not do anything, right?

Reply:
I got my answer:
```
$ dpkg -I /mnt/c/Users/lmazuel/Downloads/code_1.44.2-1587059832_amd64.deb
 new Debian package, version 2.0.
 size 62335842 bytes: control archive=2325 bytes.
     705 bytes,    14 lines      control
    2909 bytes,    73 lines   *  postinst             #!/usr/bin/env
     189 bytes,     5 lines   *  postrm               #!/bin/bash
     274 bytes,     6 lines   *  prerm                #!/usr/bin/env
 Package: code
 Version: 1.44.2-1587059832
 Section: devel
 Depends: libnotify4, libnss3 (>= 2:3.26), gnupg, apt, libxkbfile1, libsecret-1-0, libgtk-3-0 (>= 3.10.0), libxss1
 Priority: optional
 Architecture: amd64
 Maintainer: Microsoft Corporation <vscode-linux@microsoft.com>
 Homepage: https://code.visualstudio.com/
 Installed-Size: 259118
 Provides: visual-studio-code
 Conflicts: visual-studio-code
 Replaces: visual-studio-code
 Description: Code editing. Redefined.
  Visual Studio Code is a new choice of tool that combines the simplicity of a code editor with what developers need for the core edit-build-debug cycle. See https://code.visualstudio.com/docs/setup/linux for installation instructions and FAQ.
```

This means libsecret will be installed if you have vscode